hr_employee_age/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2015 Salton Massally.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Employee Age",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': "Salton Massally <smassally@idtlabs.sl>, "
+            "Odoo Community Association (OCA)",
+    "website": "http://idtlabs.sl",
```

Comment:
@Chandresh-SerpentCS Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)